Makefile

```diff
@@ -5,8 +5,16 @@
 
 MAKEFLAGS += --no-builtin-rules
 
+# Ensure the build fails if a piped command fails
+SHELL = /bin/bash
+.SHELLFLAGS = -o pipefail -c
+
+
 #### Defaults ####
 
+# Target game version. Currently only the following version is supported:
+#   us   N64 USA (default)
+VERSION ?= us
```

Comment:
Since there are versions for other platforms we should at least have `n64` somewhere in the version name
```suggestion
# Target game version. Currently only the following version is supported:
#   us   N64 USA (default)
VERSION ?= n64-us
```